README.md

```diff
@@ -2,8 +2,23 @@
 
 pg_duckdb is a Postgres extension that embeds DuckDB's columnar-vectorized analytics engine and features into Postgres. We recommend using pg_duckdb to build high performance analytics and data-intensive applications.
 
+Third party developers will be able to extend both DuckDB and the PostgreSQL integration by writing their own DuckDB extensions. Those extensions will be distributed through the DuckDB community extension repository.
+
 pg_duckdb was developed in collaboration with our partners, [Hydra](https://hydra.so) and [MotherDuck](https://motherduck.com).
 
+
+## Roadmap
+
+* Create and manage analytical tables with good performance on analytical queries
+* Broad support for Postgres types and functions within analytical context
+* Query external datasets on S3, GCP and Azure (Parquet, Iceberg and Delta Lake)
+* Join native Postgres and analytical data efficiently
+* Seamlessly run queries read and write queries against local data and MotherDuck tables
+* Allow for easy configuration and deployment of DuckDB extensions
+* Columnar indexing of native Postgres tables
+* Real-time synchronization of analytical and transactional data
```

Comment:
I don't quite know what this means or how we'd do it, I'd cut it.

Reply:
If this means near-time replication of transactional data to an analytical table, I think it should be kept. A big limitation of the extension as it is defined today is that it expects users to recreate their analytical table from the transactional table. I believe this suggests a capability of synchronizing mutations in the transactional table with the derived analytical table.

Reply:
Going to strike this and add back if/when I get clarification on functionality.